src/ticketmap/index.tsx

```diff
@@ -327,39 +339,49 @@ export default class TicketMap extends Component<Props, State> {
    * Public Methods
    */
 
-  highlightSection = (section: string) => this.toggleSectionHighlight(section, true)
+  highlightSection = (section: string): void => this.toggleSectionHighlight(section, true)
 
-  unhighlightSection = (section: string) => this.toggleSectionHighlight(section, false)
+  unhighlightSection = (section?: string): void => {
+    if (!section) {
+      this.setState({ currentHoveredSection: null })
+      return this.updateMap()
+    }
+    return this.toggleSectionHighlight(section, false)
+  }
 
-  toggleSectionHighlight = (section: string, shouldHighlight = true) => {
+  toggleSectionHighlight = (section?: string, shouldHighlight: boolean = true): void => {
     if (!section) {
```

Comment:
If the function doesn't perform properly when it receives a falsey `section`, shouldn't the parameter be required?

Reply:
Ah yes you're right, I originally had the conditional logic currently found in `unhighlightSection` residing here in the toggle function.